src/Arcus.Observability.Tests.Integration/Arcus.Observability.Tests.Integration.csproj

```diff
@@ -10,7 +10,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Arcus.Testing.Logging" Version="0.3.0" />
+    <PackageReference Include="Arcus.Testing.Logging" Version="0.4.0" />
```

Comment:
There a reason why we updated this? Fine by me that we use the most recent version, though.

Reply:
That's the only reason yes. Thought it would make sense to update it to make sure everything we did and do works with our most recent version.

Reply:
Yes, let's do this! 👍 Thx.